pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -614,7 +614,7 @@ func TestIntegrationRulerAccess(t *testing.T) {
 			desc:            "viewer request should fail",
 			client:          newAlertingApiClient(grafanaListedAddr, "viewer", "viewer"),
 			expStatus:       http.StatusForbidden,
-			expectedMessage: `You'll need additional permissions to perform this action. Permissions needed: any of alert.rules:write, alert.rules:create, alert.rules:delete`,
+			expectedMessage: `You'll need additional permissions to perform this action. Permissions needed: all of any of alert.rules:write, alert.rules:create, alert.rules:delete, alert.rules:read, folders:read`,
```

Comment:
I believe this is what the error message should be with the reordered permissions in the other suggestion.

```suggestion
			expectedMessage: `You'll need additional permissions to perform this action. Permissions needed: all of alert.rules:read, folders:read, any of alert.rules:write, alert.rules:create, alert.rules:delete`,
```